schema.graphql

```diff
@@ -371,6 +374,40 @@ type TreasuryProposal @entity {
   voteEnd: BigInt!
   "Description of the proposal"
   description: String!
+  "Votes cast for this proposal"
+  votes: [TreasuryVote!]! @derivedFrom(field: "proposal")
+  "Total weight of votes in favor"
+  forVotes: BigDecimal!
+  "Total weight of votes against"
+  againstVotes: BigDecimal!
+  "Total weight of abstaining votes"
+  abstainVotes: BigDecimal!
+  "Sum of all vote weights"
+  totalVotes: BigDecimal!
+}
+
+enum TreasuryVoteSupport @entity {
```

Comment:
Has this been deployed and is it syncing? If it is I'm pretty confused. 

I didn't think `enum` types could be entities. I would remove the `@entity` annotation.

Reply:
Good catch! You're right (see the enum documentation: [https://thegraph.com/docs/en/subgraphs/guides/enums/#what-are-enums](https://thegraph.com/docs/en/subgraphs/guides/enums/#what-are-enums)).

The subgraph was deployed, but it appears it previously failed earlier in the process due to a missing ABI. I've just fixed that issue [here](https://github.com/livepeer/subgraph/pull/175/changes/5f5dc620fa9f19686580584e7af27603deb2d087)

Reply:
@jmulq Looks like we do this in more places created an issue for it: https://github.com/livepeer/subgraph/issues/177.